pydantic/main.py

```diff
@@ -167,6 +168,9 @@ class BaseModel(metaclass=_model_construction.ModelMetaclass):
     __pydantic_validator__: ClassVar[SchemaValidator | PluggableSchemaValidator]
     """The `pydantic-core` `SchemaValidator` used to validate instances of the model."""
 
+    __pydantic_validate_call_infos__: ClassVar[Dict[str, ValidateCallInfo]] = {}  # noqa: UP006
```

Comment:
This is introduced [here](https://github.com/pydantic/pydantic/pull/10110/files#diff-8708af8d26c26644976866f974e0b810b0940ede43ff487e457057d5a1f07999R118-R123). 
But changing `Dict` to `dict` doesn't break tests in Python 3.13. I haven't tried other versions yet, but if that is the case for all versions, then surely we should use `dict`.

Reply:
Ah yes, I was wondering because some other attributes (e.g. `__pydantic_private__`) use `dict`, but those are different because we exclude them from the namespace iirc. So yes let's keep `Dict` here.